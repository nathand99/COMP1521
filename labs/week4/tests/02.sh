echo 12 | /home/cs1521/bin/spim -file fac1.s | sed -e '1d;s/n  = //'
