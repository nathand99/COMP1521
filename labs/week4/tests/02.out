n! = 479001600
